docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-inventory-management-feature-walkthrough.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Marketplace Inventory Management feature walkthrough 
+last_updated: Jul 30, 2021
+description: Merchants are product and service sellers in the Marketplace.
+template: concept-topic-template
+---
+
+## Warehouse management
+You can manage the relations between merchant and warehouse by importing the warehouse and merchant data. See [File details: merchant_stock.csv](/docs/marketplace/dev/data-import/{{ page.version }}/file-details-merchant-stock-csv.html)
+
+## Stock management
```

Comment:
why we do not mention here "merchant products"?

Reply:
Do we have an importer for updating stock of merchant products? If yes, please let me know, as I am not aware of it.

Reply:
it is a normal product stock importer. nothing marketplace-specific…

Reply:
added